kore/src/Kore/Repl/Interpreter.hs

```diff
@@ -1521,9 +1521,13 @@ graphParams gr =
             ]
         }
   where
+    nrOfNodes :: Natural -> Text.Lazy.Text
```

Comment:
Why does this use lazy text?

Reply:
This is what [`Graph.textLabel`](https://hackage.haskell.org/package/graphviz-2999.20.1.0/docs/Data-GraphViz-Attributes.html#v:textLabel) uses.